pkg/riff/commands/stream_create.go

```diff
@@ -30,7 +30,8 @@ import (
 type StreamCreateOptions struct {
 	cli.ResourceOptions
 
-	Provider string
+	Provider    string
+	ContentType string
```

Comment:
do we need to apply any validation?

Reply:
we can check the presence of `/` and check that the first slash is not also the last. I may be missing something but I don't think we can do much more.